.eslintrc.json

```diff
@@ -1,24 +1,27 @@
 {
-  "env": {
```

Comment:
I've removed the `env`, as it seems that ESLint can pretty much handle everything out of the box.